core/common/src/main/java/alluxio/conf/PropertyKey.java

```diff
@@ -356,9 +359,41 @@ public String toString() {
           .setIsHidden(true)
           .setScope(Scope.ALL)
           .build();
+  public static final PropertyKey NETWORK_CHANNEL_AUTH_TIMEOUT =
```

Comment:
@ggezer Please take a look for the next 4 properties. Actually, do you think channel means connection here? If yes, I would even rename Channel to connection. Also. I would avoid mentioning or at least give more general description in property descriptions as people may not know what GRPC is but they simply want to bump up network connection time out. what do you think?